.github/workflows/main.yml

```diff
@@ -1,29 +1,30 @@
+---
 name: Run Gradle Push
 on:
   push:
-    branches: [ main ]
+    branches: [main]
 
 jobs:
   gradle:
     runs-on: macos-latest
     steps:
-    - uses: actions/checkout@v3
-    - uses: actions/setup-java@v3
-      with:
+     - uses: actions/checkout@v4
```

Comment:
Wondering whether you add these formatting yourself or it has been done automatically from the IDE?

Reply:
I was trying to fix some crazy warning by super linter. I would reduce it back.

Reply:
Do you want me to merge it then fix it later?